modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -1,21 +1,35 @@
 import Base from 'elementor/assets/dev/js/frontend/handlers/base';
 
 export default class NestedAccordion extends Base {
+	constructor( ...args ) {
```

Comment:
I think that the constructor is redundant.
Specially if you move the

```
this.animation = null;
this.isClosing = false;
this.isExpanding = false;
```

To `getDefaultSettings`

Reply:
Can it be moved to `onInit` function?
Just thinking, should we create a singleton for the `new Ma` class call?

Reply:
Louis says we can put it in the constructor.